ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/handlers/HelpCommand.java

```diff
@@ -47,13 +55,17 @@ public void run(String... args) {
             String helpFile = PropertiesLoader.getProperty(HELP_FILE);
             String helpContent = FileUtils.readResource(helpFile);
             ballerinaShell.outputInfo(helpContent);
+
         } else {
+            String topic = String.join(" ", Arrays.copyOfRange(args, 0, args.length));
             try {
-                StringBuilder content = new StringBuilder();
-                helpProvider.getTopic(args, content);
-                ballerinaShell.outputInfo(content.toString());
+                ballerinaShell.outputInfo(DESCRIPTION_PREFIX + NEW_LINE + bbeHelpProvider.getDescription(topic)
+                        .replaceAll(TAGS, EMPTY_STRING));
+                ballerinaShell.outputInfo(URL_PREFIX + URL + topic.replaceAll(" ", "-"));
+
             } catch (HelpProviderException e) {
-                ballerinaShell.outputError(e.getMessage());
+                ballerinaShell.outputError(NEW_LINE + "Can not find topic " + topic  + NEW_LINE + NEW_LINE +
```

Comment:
Let's either quote the topic or use a colon. e.g.,
`Cannot find topic 'records'`
or 
`Cannot find topic: records`

Reply:
Updated in a1286fe5ba1e38ae8718140c18a16fadd90542e6